docs/features/ValueTuples.cs

```diff
@@ -0,0 +1,1241 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It is licensed under Apache 2.0, and the license file is called License.txt. Please copy the header from other code files.